browser/ui/android/strings/translations/android_brave_strings_en-GB.xtb

```diff
@@ -768,7 +768,7 @@ If you don't accept this request, VPN will not reconnect and your Internet conne
 <translation id="1543259266592091784">Add funds</translation>
 <translation id="3494964159182650590">Rewards Summary</translation>
 <translation id="590733288489975971">Rewards from Ads</translation>
-<translation id="207268487070188114">One-time tip</translation>
+<translation id="207268487070188114">One-Time Tip</translation>
```

Comment:
the same as above, don't touch files in `translations` folder

Reply:
updated